napalm_junos/junos.py

```diff
@@ -51,6 +51,10 @@ def open(self):
         except ConnectTimeoutError as cte:
             raise ConnectionException(cte.message)
         self.device.timeout = self.timeout
+        if hasattr(self.device, "cu"):
```

Comment:
Shouldn't be doing this in the `close()` method instead? Otherwise we could be doing:

    device.open()
    device.open()
    device.open()
    device.open()
    device.open()
    device.open()

Without any problem.